lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -137,39 +137,43 @@ def prepare_data(d, init):
     # Is there a curve displayed?
     has_curve = bool(curves)
 
-    # Get / Images
-    imagedict = {}
-    for image in axes.get_images():
-        label = image.get_label()
+    # Get ScalarMappables.
+    smdict = {}
+    for sm in [*axes.images, *axes.collections]:
```

Comment:
`sm` is a quite obscure name and not used anywhere else in the code.

I'd prefer `mappable` or `smappable`.

Reply:
done